client/src/assets/stylesheets/pages/quiz.scss

```diff
@@ -184,6 +184,12 @@
     padding: 8px !important;
   }
 
+  .homeButton {
+    margin: 40px;
+    padding:15px;
+    background-color: #3f51b5;
+  }
```

Comment:
Would it be possible to make the button fit to the text like the next and previous buttons? Right now the width seems pretty big.

![Screenshot 2023-04-03 at 9 47 03 AM](https://user-images.githubusercontent.com/54957497/229529409-7d1e93e4-78c3-48ba-ba43-8cb8eabd7e92.png)

Reply:
I'll fix it this weekend!

Reply:
It was fixed this weekend! 🤪